src/features/organizations/components/SearchResults/index.tsx

```diff
@@ -0,0 +1,113 @@
+import Fuse from 'fuse.js';
+import NextLink from 'next/link';
+import { useMemo } from 'react';
+import { Box, Typography, useTheme } from '@mui/material';
+
+import Ancestors from './Ancestors';
+import ProceduralColorIcon from '../ProceduralColorIcon';
+import { TreeItemData } from '../../types';
+
+interface SearchResultsProps {
+  flatOrgData: TreeItemData[];
+  onSwitchOrg: () => void;
+  searchString: string;
+}
+
+const SearchResults = ({
+  flatOrgData,
+  onSwitchOrg,
+  searchString,
+}: SearchResultsProps) => {
+  const theme = useTheme();
+
+  const searchResults = useMemo(() => {
+    const fuse = new Fuse(flatOrgData, {
+      keys: ['title'],
+      threshold: 0.4,
+    });
+
+    return searchString
+      ? fuse.search(searchString).map((fuseResult) => fuseResult.item)
+      : flatOrgData;
+  }, [searchString]);
```

Comment:
Searching should not be necessary here, because the `useOrgSwitcher` hook already did it.